L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -1043,10 +1047,14 @@ namespace trklet {
     //Following values are used for duplicate removal
     //Variable bin edges for 6 bins.
     std::vector<double> varRInvBins_{-rinvcut(), -0.004968, -0.003828, 0, 0.003828, 0.004968, rinvcut()};
+    //Phi bin edges for 2 bins.
+    std::vector<double> phiBins_{0, dphisectorHG()/2, dphisectorHG()};
     //Overlap size for the overlap rinv bins in DR
     double overlapSize_{0.0004};
```

Comment:
Again this should now be called rinvOverlapSize or similar.

Reply:
Done.

Reply:
@dally96 Again you replied "done" to this comment, but without changing the code. Please recheck all comments to make sure you've really implemented them.

Reply:
The changes have been implemented.